cluster/addons/fluentd-gcp/fluentd-gcp-image/Makefile

```diff
@@ -26,7 +26,7 @@
 .PHONY:	build push
 
 PREFIX=gcr.io/google_containers
-TAG = 1.29
+TAG = 1.30
```

Comment:
Let's do not bump the image right now but rather from time to time with a batch of changes.